jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/connection/KarafServerConnection.java

```diff
@@ -174,13 +175,19 @@ protected MBeanServerConnection createConnection(IServer s, String user, String
 		envMap.put("jmx.remote.credentials", new String[] { user, pass });
 		String conUrl = KarafUtils.getJMXConnectionURL(s);
 		JMXServiceURL url = new JMXServiceURL(conUrl); 
-		JMXConnector jmxc = JMXConnectorFactory.connect(url, envMap); 
-		MBeanServerConnection mbsc = jmxc.getMBeanServerConnection(); 	
+		jmxc = JMXConnectorFactory.connect(url, envMap); 
+		MBeanServerConnection mbsc = jmxc.getMBeanServerConnection(); 
 		return mbsc; 
 	}
 	
 	protected void cleanupConnection(IServer server, MBeanServerConnection connection) {
```

Comment:
yep, once thats in the sonar analysis (after PR merged and update site built) we are able to define it as false positive.